src/wakatimeScript.ts

```diff
@@ -89,3 +103,16 @@ chrome.runtime.onMessage.addListener((request: { message: string }, sender, send
     sendResponse({ html: document.documentElement.outerHTML });
   }
 });
+
+// Google Meet
+// https://meet.google.com/jzf-bwrz-djk
+if (window.location.href.startsWith('https://meet.google.com/')) {
+  // In google meet website
+  // Check every two seconds if the user is in a meeting.
+  setInterval(() => {
```

Comment:
Let's use `setTimeout` that calls itself to make sure we never set more than one interval ending up with an infinite loop.

Reply:
Ok. But I believe the `wakatimeScript.ts` only executes once per page refresh. If the user navigates to another website the script will rerun and the previous `setInterval` should get destroyed. Let me check if I can add a clear function to stay in the safe side.

Reply:
Does `wakatimeScript.ts` from a previous tab still execute if the currently focused tab changes?